libs/openchallenges/ui/src/lib/footer/footer.component.html

```diff
@@ -9,10 +9,18 @@
       <ul class="footer-link-group">
         <li><a href="/about">About</a></li>
         <li><a href="/team">Meet Our Team</a></li>
-        <!-- <li><a href="#">API</a></li> -->
         <li>
-          <a
+          <a 
+            href="https://openchallenges.io/api-docs"
+            title="Interactive documentation for OC API"
+            rel="noopener noreferrer"
+            target="_blank"
+            >API Docs</a></li>
+        <li>
+          <a 
             href="https://github.com/Sage-Bionetworks/sage-monorepo/issues/new/choose"
+            title="Share feedback about the OC app!"
```

Comment:
Avoid [special characters](https://business.scope.org.uk/article/accessibility-screen-readers-special-characters-and-unicode-symbols) in properties that text readers may heavily rely on.

```suggestion
            title="Share feedback about the OC app"
```

Reply:
> Avoid [special characters](https://business.scope.org.uk/article/accessibility-screen-readers-special-characters-and-unicode-symbols)
But I live and breathe in exclamation marks..... 😢 

But you're right - will (try to) avoid in the future.